src/changes-backwards-compatible.apib

```diff
@@ -6,7 +6,9 @@ We list all backwards-compatible additions here. These are currently available i
 
 #### Latest
 
-- The task assignee is now nullable.
+- The task assignee is now nullable: you can create / update unassigned tasks by sending null as assignee. The tasks.list and tasks.info endpoints also return null as assignee in case the task is unassigned.
+
+- On API versions older than 2019-10-09, the assignee is returned as { type: user, id: null } in case the task is unassigned.
```

Comment:
Q: It's on API versions up to and including 2019-10-09, right? (I'm assuming 2019-10-09 is the latest API version today).

Reply:
Not including. 2019-10-09 is the latest version. On 2019-10-09 and later it returns null; older than that, it returns type: user and id null.

Reply:
I thought this was a breaking change, didn't realise this is the backwards-compatible changelog. Please ignore.

Reply:
But then the contract of the assignee field changes for `2019-10-09`? So it is breaking?